hypervisor/src/kvm/mod.rs

```diff
@@ -267,10 +277,35 @@ impl vm::Vm for KvmVm {
     /// Creates a guest physical memory region.
     ///
     fn create_user_memory_region(&self, user_memory_region: MemoryRegion) -> vm::Result<()> {
+        let mut region = user_memory_region;
+
+        if (region.flags & KVM_MEM_LOG_DIRTY_PAGES) != 0 {
```

Comment:
When this is true? Does the caller ever set this flag?

Reply:
Yes, it is  expected to be set by the caller/user, just like all other members of `MemoryRegion`. 

For the current usage from the `vmm` crate, this flag when calling into `make_user_memory_region` with `log_dirty_pages = true` (check the second commit for details).